packages/golden-layout/karma.conf.cjs

```diff
@@ -25,6 +25,12 @@ module.exports = function (config) {
 
     browserify: {
       plugin: ['esmify'],
+      // ignored because it doesn't understand the scss import in GoldenLayoutThemeExport
+      configure: function (bundle) {
+        bundle.on('prebundle', function () {
+          bundle.ignore('./dist/GoldenLayoutThemeExport.js');
```

Comment:
Not sure I full understand this. The `./dist/GoldenLayoutThemeExport.js` file that's outputted has a `css` import, not an `scss` import - so what's being ignored here specifically?

Reply:
This was from the original PR, but I think it is skipping this file instead of trying to bundle it with browserify which I guess doesn't like the CSS import? Idk, but I tried removing it and the tests failed.